src/OrbitTest/OrbitTest.cpp

```diff
@@ -21,7 +21,8 @@
 // OrbitBase defines an implementation of the Orbit API for introspection. To prevent symbol clash
 // with the header-only use of the API, we wrap the include of Orbit.h in a namespace. This is only
 // needed for internal projects that link with OrbitBase, basically only OrbitTest. This should be
-// removed once we split the introspection and manual instrumentation apis TODO(b/182848566).
+// removed once we split the introspection and manual instrumentation apis
+// TODO(b/182848566).
```

Comment:
Could you move the TODO at the beginning of the paragraph?

Reply:
done